app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -91,40 +101,58 @@ private fun RecentBookmarkItem(
 ) {
     var isMenuExpanded by remember { mutableStateOf(false) }
 
-    Column(
+    Card(
         modifier = Modifier
-            .width(156.dp)
+            .width(158.dp)
             .combinedClickable(
                 enabled = true,
                 onClick = { onRecentBookmarkClick(bookmark) },
                 onLongClick = { isMenuExpanded = true }
-            )
+            ),
+        shape = cardShape,
+        backgroundColor = FirefoxTheme.colors.layer2,
+        elevation = 6.dp,
     ) {
-        Card(
+        Column(
             modifier = Modifier
                 .fillMaxWidth()
-                .height(96.dp),
-            elevation = 6.dp
+                .padding(horizontal = 16.dp)
         ) {
-            RecentBookmarkImage(bookmark)
-        }
+            Spacer(modifier = Modifier.height(16.dp))
 
-        Spacer(modifier = Modifier.height(8.dp))
+            Box(
+                modifier = imageModifier.then(
+                    Modifier.background(
+                        color = when (isSystemInDarkTheme()) {
+                            true -> PhotonColors.DarkGrey30
+                            false -> PhotonColors.LightGrey30
+                        },
+                    )
+                ),
+                contentAlignment = Alignment.Center,
+            ) {
+                RecentBookmarkImage(bookmark)
+            }
 
-        Text(
-            text = bookmark.title ?: bookmark.url ?: "",
-            color = FirefoxTheme.colors.textPrimary,
-            fontSize = 12.sp,
-            overflow = TextOverflow.Ellipsis,
-            maxLines = 1
-        )
+            Spacer(modifier = Modifier.height(8.dp))
 
-        RecentBookmarksMenu(
-            showMenu = isMenuExpanded,
-            menuItems = menuItems,
-            recentBookmark = bookmark,
-            onDismissRequest = { isMenuExpanded = false }
-        )
+            Text(
+                text = bookmark.title ?: bookmark.url ?: "",
+                color = FirefoxTheme.colors.textPrimary,
+                overflow = TextOverflow.Ellipsis,
+                maxLines = 1,
+                style = FirefoxTheme.typography.caption,
+            )
+
+            Spacer(modifier = Modifier.height(8.dp))
```

Comment:
Let's ask for updated designs without the bottom secondary text. It's possible this might end up just being 16dp with the updated changes. If that was the case, I would recommend removing the Spacer and just use padding. However, I am leaning towards just moving the Spacers here toward padding anyways. I think we prefer to use Spacers in-between elements. So, we can avoid specifying a top padding for a component.

Reply:
There isn't a secondary text in this composable, right? Was this comment meant for top sites?

Reply:
This is related to bottom text "Wallpaper accessibili.." in the recent bookmarks as seen in the designs. https://www.figma.com/file/d7l1iGabQYqnLcnQbHDGzM/Wallpaper-2.0-%2B-Onboarding-v106?node-id=5816%3A214700

<img width="359" alt="Screen Shot 2022-08-24 at 1 09 30 PM" src="https://user-images.githubusercontent.com/1190888/186480680-1ee0bb73-a872-4965-b0c5-b3070671131a.png">

Since we already know there isn't any meaning metadata to display there unless we want to display the origin url, we should ask UX to update the Figma without this caption and see if the padding will also change. Right now, we use 8dp bottom padding whereas top and horizontal are 16dp, but that 8dp bottom padding might simply be due to the extra caption content. So, I am wondering if we removed the caption, does the design spec change?

Reply:
Oh, it already has been removed from the [designs](https://www.figma.com/file/d7l1iGabQYqnLcnQbHDGzM/Wallpaper-2.0-%2B-Onboarding-v106?node-id=5458%3A209074).

<img width="331" alt="image" src="https://user-images.githubusercontent.com/87384386/186482071-291129ee-8ba7-42ab-af77-6884625d94d5.png">

Reply:
I found where the discrepancy is. The bookmark design was update in that Figma only under the section "Android - Wallpaper 2.0", whereas your screen grab and the old design is under "Android MR1 2022 Onboarding Flow"

Reply:
Oh I see, I might just be looking at the old figma that was linked in the ticket. In the new Figma, the right padding is 18dp while top and left are 16dp, so I am still seeing some inconsistency. Let's just flag UX for more changes in the Figma to get it pixel perfect.

Reply:
<img width="643" alt="Screen Shot 2022-08-29 at 3 04 45 PM" src="https://user-images.githubusercontent.com/87550321/187307109-6aa4230a-949a-4ac7-940c-d407391b7a0d.png">

Updated!

Reply:
Looks great Noah.